ipt-enable-logs/scripts/ipt-enable-logs/main.go

```diff
@@ -51,12 +51,12 @@ func getActionName(action string) string {
 }
 
 func getRuleName(rule string, commentNr int) string {
-	rule = strings.Replace(rule, "PREROUTING", "PRER", 1)
-	rule = strings.Replace(rule, "POSTROUTING", "POSTR", 1)
-	rule = strings.Replace(rule, "HOOK", "HK", 1)
-	rule = strings.Replace(rule, "USER", "U", 1)
+	ruleName := strings.Replace(rule, "PREROUTING", "PRER", 1)
+	ruleName = strings.Replace(ruleName, "POSTROUTING", "POSTR", 1)
+	ruleName = strings.Replace(ruleName, "HOOK", "HK", 1)
+	ruleName = strings.Replace(ruleName, "USER", "U", 1)
 	if commentNr != 0 {
-		rule = fmt.Sprintf("%s-%d", rule, commentNr)
+		ruleName = fmt.Sprintf("%s-%d", ruleName, commentNr & 0xFFFF)
```

Comment:
I think this should be`0xFFFFFFFF` - rather than `0xFFFF`. You need two hexadecimal characters for each of the 4 bytes in a 32-bit number.
```suggestion
		ruleName = fmt.Sprintf("%s-%d", ruleName, commentNr & 0xFFFFFFFF)
```